migrations/20210805002345-create-user-confirmation.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  up(queryInterface, Sequelize) {
```

Comment:
tablo oluşturma ve `users` tablosuna kolon ekleme aynı hedefe hizmet ettiği için aynı dosya içinde olmalı çünkü eğer biz bu şekilde bir email doğrulaması yapmaktan vaz geçersek işlemler aynı dosyada olursa tek `undo` işlemiyle eski halimize dönebiliriz bu yüzden `up` ve `down`  fonksiyonlarını `migrations/20210729004806-create-email-confirmation-tokens.js` içine eklemelisin